rclcpp/test/rclcpp/allocator/test_allocator_common.cpp

```diff
@@ -23,7 +23,7 @@ TEST(TestAllocatorCommon, retyped_allocate) {
   void * untyped_allocator = &allocator;
   void * allocated_mem =
     rclcpp::allocator::retyped_allocate<std::allocator<char>>(1u, untyped_allocator);
-  ASSERT_NE(nullptr, allocated_mem);
+  ASSERT_TRUE(nullptr != allocated_mem);
```

Comment:
Mind adding a quick comment why these are changed so they don't get reverted?

Reply:
Good point.  Added comments in 8594ab3